src/mahoji/commands/steal.ts

```diff
@@ -100,7 +110,7 @@ export const stealCommand: OSBMahojiCommand = {
 			return 'This NPC/Stall is missing variable respawnTime.';
 		}
 
-		const maxTripLength = calcMaxTripLength(user, 'Pickpocket');
+		const maxTripLength = (stealable.name === 'Wealthy Citizen' ? 2 : 1) * calcMaxTripLength(user, 'Pickpocket');
```

Comment:
a 2x higher max trip length, are they really _that_ afk?

Reply:
Given that regular pickpocketing is very click intensive compared to regular woodcutting, I figured the comparison was valid. In game, this is considered a 'second monitor' activity, like redwoods or amethysts.